lib/pana.dart

```diff
@@ -46,38 +44,32 @@ class PackageAnalyzer {
   Future<Summary> inspectPackage(
     String package, {
     String version,
-    String packageDir,
     bool keepTransitiveLibs: false,
   }) async {
-    var toolProblems = <ToolProblem>[];
-    var sdkVersion = _dartSdk.version;
-    log.info("SDK: $sdkVersion");
-
-    log.info("Package: $package");
+    _logEnv();
+    log.info("Downloading package $package ${version ?? 'latest'}");
+    var pkgInfo = await _pubEnv.getLocation(package, version: version);
+    final packageDir = pkgInfo.location;
+    return _inspect(packageDir, keepTransitiveLibs);
+  }
 
-    Version ver;
-    if (version != null) {
-      ver = new Version.parse(version);
-      log.info("Version: $ver");
-    }
+  Future<Summary> inspectDir(String packageDir,
+      {bool keepTransitiveLibs: false}) async {
+    _logEnv();
```

Comment:
The next call to `_inspect` also calls `_logEnv` – redundant?

Reply:
Good catch! The intent here is to log env information before starting to download the package, and that's reason it is duplicated in both method. Shall I just move both `_inspect`?

Reply:
Well, it's already there 😄

Reply:
done